### PR TITLE
3 feature docs

### PR DESCRIPTION
**Pull request by corysutyak**

### CODE REVIEW

File: docs/features/page_types/article_pages.rst [hunk 1]
```diff
@@ -0,0 +1,33 @@
+Article Pages
+=============
+
+Create a blog
+
+Usage
+-----
+
+First start by creating an "Article Landing Page" and then add any number of "Article Page"'s as children of the "Article Landing Page".  Each child page here represents an individual article.  Add content to these pages as you would for a normal "Web Page".
```
Comment (tomdyson) [inline docs/features/page_types/article_pages.rst]: "Article Page"'s -> "Article Page"s

File: docs/features/page_types/article_pages.rst [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+
+Article Landing Page
+--------------------
+
+Layout Tab
+~~~~~~~~~~
+
+* **Show Images**:  If toggled on, this landing page will show the cover images for it's children articles.
```
Comment (tomdyson) [inline docs/features/page_types/article_pages.rst]: it's -> its (and 3 lines below)

File: docs/features/page_types/article_pages.rst [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+Article Page
+------------
+
+Content Tab
+~~~~~~~~~~~
+
+* **Caption**: The caption you want to be displayed under your article's title.
+* **Author**: The user you want to tie the article to.
+* **Display Author As**: The name you want the author to appear as.  This will default to the Author's name if not defined.
```
Comment (tomdyson) [inline docs/features/page_types/article_pages.rst]: Author's -> author's

File: docs/features/page_types/article_pages.rst [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+* **Display Publish Date**: The publish date you want to be shown on your article.  This is seperate from the actual publish date which is determined when the page is published.
```
Comment (tomdyson) [inline docs/features/page_types/article_pages.rst]: seperate -> separate

File: docs/features/page_types/event_pages.rst [hunk 5]
```diff
@@ -1,36 +1,40 @@
...
 for each event, recurring events, or all events.
 
-
 Usage
 -----
 
-If events are implemented on your site (see instructions below), first start by creating an
-"Event Landing Page" (may be named differently on your specific website). Add content to this
-page as usual. Under the **Layout** tab, you can choose a few options:
+First start by creating an "Event Landing Page" and then add any number of "Event Page"'s as children of the "Event Landing Page". Each child page here represents an individual event. Events can be one time, or recurring, similar to Outlook or other calendar software. Add content to these pages as you would for a normal "Web Page".
```
Comment (tomdyson) [inline docs/features/page_types/event_pages.rst]: "Event Page"'s -> "Event Page"s

File: docs/features/page_types/index.rst [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+    event_pages
+    form_pages
+    location_pages
+    web_pages
+
+
+Design Philosophy
+-----------------
+
+Pages for your Codered CMS site use a "Parent-Child" relationship.  A parent page is any page that is an ancestor of other pages in the site's tree structure.  A child page is any page that is a descendant of another page in the site's tree structure.  A lot of specific site functionality is broken up into these "Parent-Child" relationships.  For example, if you want to add a blog to your site, you would add an "Article Landing Page", which would have your "Article Page"'s as descendants.
```
Comment (tomdyson) [inline docs/features/page_types/index.rst]: "Article Page"'s -> "Article Page"s

File: docs/features/page_types/location_pages.rst [hunk 7]
```diff
@@ -0,0 +1,93 @@
+Location Pages
+===================
+
+Create location pages for individual locations. These could be stores, distributors, facilities, etc. Placing these location pages under a location landing page aggregates them using an interactive Google Map.
+
+Usage
+-----
+
+First start by creating a "Location Landing Page" and then add any number of "Location Page"'s as children of the "Location Landing Page". Each child page here represents a location that will have it's own page and show up in it's parent google map.  Add content to this page as you would for a normal Web Page.
```
Comment (tomdyson) [inline docs/features/page_types/location_pages.rst]: it's -> its (*2)
google -> Google

File: docs/features/page_types/location_pages.rst [hunk 8]
```diff
@@ -0,0 +1,93 @@
...
+
+Location Landing Page
+---------------------
+
+Layout Tab
+~~~~~~~~~~
+
+* **Center Latitude**: The latitude you want the google map to center on.
+* **Center Longitude**: The longitude you want the google map to center on.
+* **Zoom**: The zoom level you want hte google map to default to.  This requires an API key to use zoom. The zoom values can be between 1-20.  1: World, 5: Landmass continent, 10: City, 15: Streets, 20: Buildings
```
Comment (tomdyson) [inline docs/features/page_types/location_pages.rst]: hte -> the
google -> Google (a few times)

**Comment by corysutyak** [comment]
Thanks Tom, I appreciate the extra set of eyes.

File: docs/features/page_types/index.rst [hunk 9]
```diff
@@ -14,7 +14,7 @@ Page Types
 Design Philosophy
 -----------------
 
-Pages for your Codered CMS site use a "Parent-Child" relationship.  A parent page is any page that is an ancestor of other pages in the site's tree structure.  A child page is any page that is a descendant of another page in the site's tree structure.  A lot of specific site functionality is broken up into these "Parent-Child" relationships.  For example, if you want to add a blog to your site, you would add an "Article Landing Page", which would have your "Article Page"'s as descendants.  
+Pages for your Codered CMS site use a "Parent-Child" relationship.  A parent page is any page that is an ancestor of other pages in the site's tree structure.  A child page is any page that is a descendant of another page in the site's tree structure.  A lot of specific site functionality is broken up into these "Parent-Child" relationships.  For example, if you want to add a blog to your site, you would add an "Article Landing Page", which would have your "Article Page"s as descendants.
```
Comment (vsalvino) [inline docs/features/page_types/index.rst]: should be capitalized as "CodeRed"

File: docs/features/snippets/index.rst [hunk 10]
```diff
@@ -0,0 +1,14 @@
...
+    :maxdepth: 1
+
+    carousels
+    classifiers
+    content_walls
+    footers
+    navigation_bars
+    reusable_content
+
+Codered CMS follows the Wagtail philosophy when it comes to snippets (viewable here<https://docs.wagtail.io/en/latest/topics/snippets.html>).
```
Comment (vsalvino) [inline docs/features/snippets/index.rst]: Should be capitalized "CodeRed"

**Comment by vsalvino** [comment]
Thanks for the proofreading, @tomdyson

**Pull request merged**